docs/pallas/gpu/pipelining.md

```diff
@@ -0,0 +1,325 @@
+---
+jupyter:
+  jupytext:
+    formats: ipynb,md
+    main_language: python
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.16.4
+  kernelspec:
+    display_name: Python 3
+    name: python3
+---
+
+(pallas_mgpu_pipelining)=
+<!-- #region id="bJ5yuIr-M0x0" -->
+
+## Mosaic GPU Pipelining
+
+This guide covers software pipelining using the Mosaic GPU backend for Pallas.
+
+For a general overview of the pipelining API in Pallas, we recommend that users first read {ref}`pallas_software_pipelining`. Pipelining in Pallas is programmed explicitly. For those who are familiar with Triton, this is a significant difference in programming model because in Triton, pipelining is an optimization that is done automatically by the compiler.
+
+<!-- #endregion -->
+
+```python id="dGAa3iO5DoRT"
+import jax
+from jax import lax
+from jax import numpy as jnp
+from jax.experimental.pallas import mosaic_gpu as plgpu
+from jax.experimental import pallas as pl
+import numpy as np
+```
+
+<!-- #region id="Pv9j90hVyswo" -->
+
+### Pipelining with Mosaic GPU
+
+The recommended approach to pipeline using Mosaic GPU is to use the {func}`jax.experimental.pallas.mosaic_gpu.emit_pipeline` function to pipeline over sequential loops (and to use `plgpu.kernel` to partition the problem in parallel over the CUDA grid). `emit_pipeline` follows a similar API as `pl.pallas_call` except it exposes a few additional GPU-specific options.
```

Comment:
nit: just say `plgpu.emit_pipeline` + perhaps add a note block at the top reminding people that `plgpu` is `jax.experimental.pallas.mosaic_gpu`?

Reply:
Done.